Assets/MRTK/Providers/OpenXR/Scripts/OpenXRDeviceManager.cs

```diff
@@ -221,6 +221,8 @@ protected override Type GetControllerType(SupportedControllerType supportedContr
                     return typeof(MicrosoftMotionController);
                 case SupportedControllerType.HPMotionController:
                     return typeof(HPReverbG2Controller);
+                case SupportedControllerType.OculusTouch:
+                    return typeof(OculusController);
```

Comment:
Looks like this code isn't ever using the new `OculusController` type yet, since `GetCurrentControllerType` (and `GetInputSourceType`, but less importantly since that returns `InputSourceType.Controller` in the base call anyway) don't handle Oculus yet. It's probably just using `SupportedControllerType.WindowsMixedReality` at the moment

Reply:
Added ifdefs for the android platform